teamscale_precommit_client/git_utils.py

```diff
@@ -1,17 +1,18 @@
 from __future__ import absolute_import
-from __future__ import unicode_literals
 from __future__ import print_function
+from __future__ import unicode_literals
 
 import os
-from pygit2 import Repository, GIT_STATUS_WT_DELETED, GIT_STATUS_WT_MODIFIED, GIT_STATUS_WT_RENAMED, \
-    GIT_STATUS_WT_TYPECHANGE, GIT_STATUS_INDEX_NEW, GIT_STATUS_INDEX_MODIFIED, GIT_STATUS_INDEX_DELETED, \
-    discover_repository
 from io import open
 
-_STATI_CONSIDERED_FOR_PRECOMMIT = GIT_STATUS_INDEX_NEW | GIT_STATUS_INDEX_MODIFIED | GIT_STATUS_WT_MODIFIED | \
-                                  GIT_STATUS_WT_RENAMED | GIT_STATUS_WT_TYPECHANGE
+from git import Repo, InvalidGitRepositoryError
+
+# [M]odified, [A]dded, [C]opied, [T]ype changed, [R]enamed (R092 should be R according to
+# https://gitpython.readthedocs.io/en/stable/reference.html#git.diff.DiffIndex, but testing it locally gave R092
+_CHANGE_TYPES_CONSIDERED_FOR_PRECOMMIT = ['M', 'A', 'C', 'T', 'R092']
```

Comment:
should we include 'R' as well, just to be safe? maybe updating the gitpython dependency will fix this at some point and then we probably won't remember to change R092 to R here

Reply:
:+1: